inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupServiceImpl.java

```diff
@@ -155,7 +155,9 @@ public String save(InlongGroupRequest request, String operator) {
             throw new BusinessException(ErrorCodeEnum.GROUP_DUPLICATE);
         }
 
-        request.setEnableZookeeper(enableZookeeper ? InlongConstants.ENABLE_ZK : InlongConstants.DISABLE_ZK);
+        if (request.getEnableZookeeper() == null) {
```

Comment:
```suggestion
        if (request.getEnableZookeeper() == null) {
```  request.setEnableZookeeper = request.getEnableZookeeper() == null ? enableZokeeper : request.getEnableZookeeper();